include/arkode/arkode_mristep.h

```diff
@@ -170,7 +179,7 @@ SUNDIALS_EXPORT void MRIStepCoupling_Write(MRIStepCoupling MRIC, FILE* outfile);
  * User-Supplied Function Types
  * ------------------------------ */
 
-typedef int (*MRIStepPreInnerFn)(sunrealtype t, N_Vector* f, int nvecs,
+typedef int (*MRIStepPreInnerFn)(sunrealtype t, N_Vector1d f, int nvecs,
```

Comment:
`doc/shared/sundials/Types.rst` introduced `sunrealtypeNd` and `sunindextypeNd`, but it never mentioned `N_Vector1d` -- should that be added?